Maquetado/scss/details.scss

```diff
@@ -13,21 +13,30 @@
   max-height: 415px;
   max-width: 1000px;
   padding: 20px;
+  position: relative;
   width: 100%;
 }
 
 .book-info {
   margin: 0 50px;
+  width: 600px;
 }
 
 .book-img {
   height: 100%;
   width: 260px;
 }
 
+.book-badge {
+  position: absolute;
+  transform: rotate(20deg) scale(0.2) translate(-120px,-1200px);
```

Comment:
`translate(-120px,-1200px);` it's too static. So i recommend you change add `right: 0; top: 0;` and later replace `translate(-10px,-10px);` or something similar 😉

Reply:
Yes, we where having some troubles with that, because the scale modifies the position